app/invocation/cache_requests_card.tsx

```diff
@@ -374,6 +375,7 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
   private getCacheMetadata(scorecardResult: cache.ScoreCard.IResult) {
     const digest = scorecardResult.digest;
     const remoteInstanceName = this.props.model.getRemoteInstanceName();
+    const ct = toResourceCacheType(scorecardResult.cacheType);
```

Comment:
nit: for frontend code we usually avoid short abbreviations unless they're common

Reply:
Done